src/main/java/io/zold/api/RtTransaction.java

```diff
@@ -209,11 +231,11 @@ public String bnf() throws IOException {
     @Override
     public String details() throws IOException {
         final String dtls = new UncheckedText(
-            new IoCheckedScalar<>(
-                new ItemAt<>(
-                    // @checkstyle MagicNumber (1 line)
-                    5, new SplitText(this.transaction, ";")
-                )
+            new IoCheckedScalar<String>(
+                () -> new ItemAt<Text>(
```

Comment:
@paulodamaso same here, we can avoid the lambda here and re-wrapping of `String` into `IoCheckedScalar<String>` by changing `transaction` to `IoCheckedScalar`.